charts/metering-ci/values.yaml

```diff
@@ -14,4 +14,6 @@ hdfsNamenodeMemory: null
 hiveMetastoreStorageSize: null
 hiveMetastoreMemory: null
 hiveMetastoreCpu: null
+prestoMemory: null
+prestoCpu: null
```

Comment:
would `prestoCPU` read better?

Reply:
Mostly going for consistency with the other fields. Not really worried about that since it's only used by CI.